src/__tests__/__snapshots__/LinkConnect.test.tsx.snap

```diff
@@ -12,11 +12,7 @@ exports[`LinkConnect Component renders correctly when linkToken is provided 1`]
   <View
     injectedJavaScript="
     window.meshSdkPlatform='reactNative';
-    window.meshSdkVersion='2.0.1';
-    if(window.parent){
-      window.parent.meshSdkPlatform='reactNative';
-      window.parent.meshSdkVersion='2.0.1';
-    }
+    window.meshSdkVersion='2.0.2';
```

Comment:
Are you going to change this snapshot each time the version is changing? Can we use some global variable here?

Reply:
actually this one is bind with the package.json file, so each time we update the version it should be changing the version in utility as well. 

Here' we always will get the version intended. (and snapshot is updated automatically)